PhysicsTools/NanoAOD/python/nano_eras_cff.py

```diff
@@ -23,3 +24,5 @@
 )
 
 run2_egamma = (run2_egamma_2016 | run2_egamma_2017 | run2_egamma_2018)
+
+run3_nanoAOD_ANY = (run3_common | run3_nanoAOD_122 | run3_nanoAOD_124 | run3_jme_Winter22runsBCDEprompt)
```

Comment:
why does it have be nano-version specific? Can't one just use `run3_common` in `nano_cff` ?

Reply:
for reference, the `_122`, `_124`, ... suffix is for the MINI input version. there fore run3_common would not function here

Reply:
I don't see why.